packages/@aws-cdk/aws-ecs-patterns/lib/base/network-load-balanced-service-base.ts

```diff
@@ -86,6 +86,13 @@ export interface NetworkLoadBalancedServiceBaseProps {
    */
   readonly loadBalancer?: NetworkLoadBalancer;
 
+  /**
+   * Listener port of the network load balancer that will serve traffic to the service.
+   *
+   * @default - defaults to 80 if not specified
```

Comment:
Please provide the default as only the value.

@default 80

Reply:
fixed